docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -0,0 +1,264 @@
+
+You can get started by looking at existing storybook stories:
+
+- [Button examples](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-button/stories/Button)
+- [Avatar examples](https://github.com/microsoft/fluentui/tree/master/packages/react-components/react-avatar/src/stories/Avatar)
+
+### Additional documentation
+
+- Documentation comments in code for all public APIs
+- A `README.md` file in the component package
+- Initial migration guide from v8 and v0
+
+<a id="unstable-release"></a>
+
+## Preview Release
+
+Once the component is functional, it can be released as an unstable preview. This will allow both `designers` to review the built component and `partners` to try it out and give early feedback.
+
+Tests are not required to be completed before releasing the component as unstable, but it is recommended to at least add conformance tests and some basic unit tests.
+
+Run the following command (replace `MYCOMPONENT` with the appropriate package name):
+
+```bash
+yarn nx generate prepare-initial-release --project @fluentui/react-MYCOMPONENT-preview --phase=preview
+```
+
+# Phase 3: Validation and Stable Release
+
+Once the component has been released as unstable, you should perform additional validation and complete any tests and documentation that was not finished before.
+
+## Tests
+
+- **Conformance tests**
+  - Call `isConformant` from your components' tests.tsx file.
+- **Unit tests**
+  - Render your component with [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro) for the tests.
+  - Use [queries](https://testing-library.com/docs/queries/about) like `getByRole` to find elements in the rendered DOM tree, and check that their attributes are as expected.
+  - Avoid using snapshot tests.
+  - When deciding what tests to write, generally the tests should cover all functionality in your component's state hook (`use{Component}`)
+  - Follow examples from other components for more guidance, such as [Button.test.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/src/components/Button/Button.test.tsx)
+- **Visual Regression Tests**
+  - VR tests ensure that the components visuals do not change unexpectedly.
+  - Add stories to the `vr-tests-react-components` app, for example [Button.stories.tsx](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Button.stories.tsx)
+  - When deciding what tests to write, generally the tests should cover all states in your component's styling hook (`use{Component}Styles`).
+- **Bundle Size Tests**
+  - Example: [Button.fixture.js](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/bundle-size/Button.fixture.js)
+- **Performance Tests**
```

Comment:
this is not relevant as it uses obsolete perf framework solution using archaic chromium. lets remove it or explicitly mention that this is no longer required

Reply:
Removed :-)